tests/values.yaml

```diff
@@ -1,11 +1,11 @@
 nodes:
   - name: 'node-1'
     msp_id: 'MyPeer1MSP'
-    address: 'http://substra-backend.node-1'
+    address: 'http://substra-backend.node-1.com'
     user: 'node-1'
     password: 'node-1pwd'
   - name: 'node-2'
     msp_id: 'MyPeer2MSP'
-    address: 'http://substra-backend.node-2'
+    address: 'http://substra-backend.node-2.com'
```

Comment:
I haven't tested this yet but does it really work? I think this is the only place I've seen urls in ".node-X.com". I checked the backend in the users branch and the skaffold file uses `http://substra-backend.node-1`.

Reply:
Yes, I'm updating it currently :)